lms/envs/common.py

```diff
@@ -1248,9 +1248,9 @@
     'js/vendor/jquery-ui.min.js',
     'js/vendor/jquery.qtip.min.js',
     'js/vendor/jquery.ba-bbq.min.js',
-    'js/vendor/afontgarde/modernizr.fontface-generatedcontent.js',
-    'js/vendor/afontgarde/afontgarde.js',
-    'js/vendor/afontgarde/edx-icons.js'
+    'pattern-library/js/modernizr.custom.js',
```

Comment:
@clrux You need to verify with @andy-armstrong that you can make these changes. I believe this was tried once before and did not work.